libs/domain/product/src/services/adapter/default-product-list.adapter.ts

```diff
@@ -55,13 +55,20 @@ export class DefaultProductListAdapter implements ProductListAdapter {
   }
 
   get(qualifier: ProductListQualifier): Observable<ProductList> {
+    const include = [
+      ApiProductModel.Includes.AbstractProducts,
+      ApiProductModel.Includes.ConcreteProducts,
+      ApiProductModel.Includes.ConcreteProductImageSets,
+      ApiProductModel.Includes.ConcreteProductPrices,
+      ApiProductModel.Includes.ConcreteProductAvailabilities,
+      ApiProductModel.Includes.Labels,
+    ].filter((type, index, arr) => arr.indexOf(type) === index);
```

Comment:
I am wondering, why do we need this filter?

Reply:
thanks!